Dockerfile

```diff
@@ -26,6 +26,7 @@ COPY test/ ./test/
 COPY fixtures/ ./fixtures/
 
 # Check code style and run static analysis along with tests
-RUN flake8 . && \
+# Ignore W605 errors (\* as invalid escape sequence) for sphinx doc builds.
```

Comment:
I don’t see that in the docstring?

Reply:
https://github.com/AxiomExergy/influx-client/blob/master/influx/__init__.py#L672 this was the problem docstring